src/AlgAssAbsOrd/Conjugacy/Conjugacy.jl

```diff
@@ -542,18 +557,24 @@ end
 function _basis_of_integral_commutator_algebra(As::Vector{fmpq_mat},
                                                Bs::Vector{fmpq_mat})
```

Comment:
This function is not tested I think. Since you changed it, could you add a test to make sure it still works?